jobs/experiments-monitoring-data-export/experiments_monitoring_data_export/export.py

```diff
@@ -0,0 +1,322 @@
+"""Exports experiment monitoring data to GCS as JSON."""
+
+import random
+import string
+from argparse import ArgumentParser
+from datetime import datetime, timedelta
+from functools import partial
+from multiprocessing import Pool
+
+import smart_open
+from google.cloud import bigquery, storage
+
+parser = ArgumentParser(description=__doc__)
+parser.add_argument(
+    "--source_project",
+    "--source-project",
+    default="moz-fx-data-shared-prod",
+    help="Project containing views to be exported",
+)
+parser.add_argument(
+    "--destination_project",
+    "--destination-project",
+    default="moz-fx-data-experiments",
+    help="Project with bucket data is exported to",
+)
+parser.add_argument(
+    "--bucket", default="mozanalysis", help="GCS bucket data is exported to"
+)
+parser.add_argument(
+    "--gcs_path", "--gcs-path", default="monitoring", help="GCS path data is written to"
+)
+parser.add_argument(
+    "--datasets",
+    help="Experiment monitoring datasets to be exported",
+    nargs="*",
+    default=[],
+)
+
+
+def get_active_experiments(client, date, dataset):
+    """
+    Determine active experiments.
+
+    Experiments are considered as active if they are currently live or have been live
+    within the past 14 days.
+    Returns the experiment slug and experiment start date.
+    """
+    job = client.query(
+        f"""
+        SELECT DISTINCT experiment, start_date
+        FROM `{dataset}`
+        LEFT JOIN (
+            SELECT
+                start_date,
+                end_date,
+                normandy_slug
+            FROM `moz-fx-data-experiments.monitoring.experimenter_experiments_v1`
+        )
+        ON experiment = normandy_slug
+        WHERE TIMESTAMP_ADD(TIMESTAMP(end_date), INTERVAL 14 DAY) >= TIMESTAMP('{date}')
+        """
+    )
+
+    result = job.result()
+    return [
+        (row.experiment, datetime.combine(row.start_date, datetime.min.time()))
+        for row in result
+    ]
+
+
+def view_to_tmp_table(view, tmp_dataset, source_project):
+    """Write the view data into a temporary table and return the destination table."""
+    client = bigquery.Client(source_project)
+    tmp = "".join(random.choices(string.ascii_lowercase, k=8))
+    tmp_table_name = view.split(".")[-1] + "_" + tmp
+    tmp_table = f"{source_project}.{tmp_dataset}.{tmp_table_name}"
+
+    job_config = bigquery.QueryJobConfig(destination=tmp_table)
+    job_config.clustering_fields = ["experiment"]
+
+    sql = f"""
+        SELECT
+            `time`,
+            branch,
+            experiment,
+            value
+        FROM {view}
+    """
+
+    query_job = client.query(sql, job_config=job_config)
+    query_job.result()
+    return tmp_table
```

Comment:
This is the main difference compared to the existing export script in bigquery-etl